### DISCUSSION TITLE
Credhub-cli dependency update

### DISCUSSION THREAD

**Discussion by lsangelov** [concourse/concourse#8209] [Help & Support] [answered]
In our team we execute WhiteSource scans for Concourse versions we use. 
In the latest concourse releases a pretty old( almost 3 years old) version of [credhub-cli](https://github.com/concourse/concourse/blob/master/go.mod#L7) is used
which is resembled to version [2.5.0](https://github.com/cloudfoundry/credhub-cli/releases/tag/2.5.0) of credhub-cli.

This version 2.5.0 of credhub-cli has 13 High and 1 medium vulnerabilities reported.

Do you plan to update the used version of credhub-cli in near future?
Is there a process to request such update?

**Comment by taylorsilva** [comment, marked as answer]
For others, stuff like this can be emailed to security@concourse-ci.org

**Reply by taylorsilva** [reply]
Addressed by: https://github.com/concourse/concourse/issues/8215

**Reply by lsangelov** [reply]
Thanks for the fast reaction
